packages/odyssey-stylelint/src/index.js

```diff
@@ -13,13 +13,13 @@
 module.exports = {
   plugins: [
     require.resolve('./rules'),
-    'stylelint-order',
     'stylelint-scss'
   ],
   extends: [
     'stylelint-config-recommended',
     'stylelint-config-standard',
```

Comment:
`stylelint-config-recommended` extends `stylelint-config-standard` under the hood so we can drop line 21 and the dependency from our `package.json` as well as it will be installed transitively.

Reply:
great catch - removed!

Reply:
thanks for catching the fact I wrote that backward in my comment! updated to be more clear